<launch>
  <!-- Launch MoveIt! -->
  <include file="$(find motoman_sia5d_moveit_config)/launch/moveit_planning_execution.launch"/>
  
  <!-- fake_joint_driver_node -->
  <node name="fake_joint_driver" pkg="fake_joint_driver" type="fake_joint_driver_node"/>

  <!-- Load joint trajecotory controller -->
  <rosparam file="$(find jog_controller)/config/motoman_sia5d_controllers.yaml" />
  <node name="controller_spawner"
	pkg="controller_manager" type="spawner" respawn="false"
	args="joint_state_controller joint_trajectory_controller"/> 
  
  <node name="jog_joint_node" pkg="jog_controller" type="jog_joint_node">
    <rosparam command="load"
	      file="$(find jog_controller)/config/motoman_sia5d_jog_joint.yaml"/>
  </node>

  <node name="jog_frame_node" pkg="jog_controller" type="jog_frame_node">
    <param name="controller_name" value="joint_trajectory_controller"/>
    <param name="target_link" value="link_t"/>
  </node>

</launch>
